webapp/components/wpt-flags.js

```diff
@@ -46,7 +46,7 @@ Object.defineProperty(wpt, 'ClientSideFeatures', {
       'showTestRefURL',
       'structuredQueries',
       'searchPRsForDirectories',
-      'webPlatformTestsLive',
```

Comment:
Changing this feature name will temporarily un-launch the feature (we need to change it in data store too).